API/pinning-api/README.md

```diff
@@ -0,0 +1,91 @@
+Pin API
```

Comment:
The folder is called `pinning-api`, so this probably should be called `Pinning API`